### ISSUE TITLE
Part selector overlaps problem text

### ISSUE THREAD

**Issue by gluecia** [KalenShamy/AdventOfRacket#3]
For accessibility, the part selector should probably be placed below the problem description, or there should be a more clear background around it to show that it's distinct from the rest of the text.

<img width="364" height="116" alt="showing the part picker overlap" src="https://github.com/user-attachments/assets/82e38361-7e7e-46a0-828b-ed3e4cc4f4bf" />
